test/Microsoft.Build.Sql.Tests/TestData/CodeAnalyzerSample/CodeAnalyzerSample.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>netstandard2.1</TargetFrameworks>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_AddAnalyzersToOutput</TargetsForTfmSpecificContentInPackage>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SqlServer.DacFx" Version="162.3.566" PrivateAssets="all" />
+  </ItemGroup>
+  <Target Name="GetTargetPath" />
+  <Target Name="_AddAnalyzersToOutput">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(OutputPath)\CodeAnalyzerSample.dll" PackagePath="analyzers/dotnet/cs" />
```

Comment:
Why this uncommon location? How about "tools" ?

Reply:
This is the [path Nuget natively resolves analyzers from](https://learn.microsoft.com/nuget/guides/analyzers-conventions#analyzers-path-format)

Reply:
Ok, but this is not a roslyn analyzer and the target language is not "cs" ...

Reply:
Yeah I can understand that, my motivation was to keep it similar to C# and use Nuget's package resolution logic. We did build in the extensibility via `SqlCodeAnalysisAssemblyPaths` so if there is a growing demand for another location, we can easily hook that up.

Reply:
> Nuget's package resolution logic

Fair, if that adds value for you.

But maybe just use `analyzers\dotnet\` ??